contracts/GenArt721CoreV3.sol

```diff
@@ -901,6 +961,7 @@ contract GenArt721CoreV3 is
         external
         onlyUnlocked(_projectId)
         onlyArtistOrAdminACL(_projectId, this.updateProjectScript.selector)
+        onlyNonEmptyString(_script)
```

Comment:
nit: should we also be using `onlyValidProjectId ` since adminACL can call this?

Reply:
ah, yes this makes sense! TY